web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/assets/css/views-basic.css

```diff
@@ -0,0 +1,14 @@
+.views-basic--params {
```

Comment:
Could you add a @todo to this file to consider moving the admin styles to the yalesites admin theme? We may not change anything, but I want to be able to search for 'todo's later. In February we are going to have a print on theming the backend interface and we will want to decide where these sorts of bespoke CSS files should exist.

Reply:
Added.